### PR TITLE
[12.0][MIG] stock_removal_location_by_priority

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Standard migration.

### CODE REVIEW

**Review by LoisRForgeFlow — APPROVED** [review]
:+1:

File: stock_removal_location_by_priority/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Stock Removal Location by Priority",
+    "summary": "Establish a removal priority on stock locations.",
+    "version": "12.0.1.0.0",
+    "author": "Eficent, "
+              "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/stock-logistics-warehouse",
+    "category": "Warehouse Management",
```
Comment (rousseldenis) [inline stock_removal_location_by_priority/__manifest__.py]: @mreficent Change this to 'Warehouse' to get the good category in OCA Shop

File: stock_removal_location_by_priority/models/stock_quant.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+
+
+class StockQuant(models.Model):
+    _inherit = 'stock.quant'
+
+    removal_priority = fields.Integer(
+        related='location_id.removal_priority',
+        readonly=True, store=True,
```
Comment (rousseldenis) [inline stock_removal_location_by_priority/models/stock_quant.py]: You can remove readonly

**Review by rousseldenis — CHANGES_REQUESTED** [review]
LGTM. Two comments

File: stock_removal_location_by_priority/__manifest__.py (hunk 1 above)
Comment (MiquelRForgeFlow) [reply stock_removal_location_by_priority/__manifest__.py]: good!

File: stock_removal_location_by_priority/models/stock_quant.py (hunk 2 above)
Comment (MiquelRForgeFlow) [reply stock_removal_location_by_priority/models/stock_quant.py]: done!

**Review by rousseldenis — APPROVED** [review]

**Pull request closed without merging**